offer-manager-domain/src/test/groovy/life/qbic/business/productitem/ProductItemViewDtoSpec.groovy

```diff
@@ -0,0 +1,19 @@
+package life.qbic.business.productitem
+
+import spock.lang.Specification
+
+/**
+ * <p>Checks for equality and hashcode to work</p>
+ *
+ * @since 1.1.0
+ */
+class ProductItemViewDtoSpec extends Specification {
+
+    def "equality is correct for equal product item vies"() {
+        //TODO
+    }
+
+    def "different objects are not equal"() {
+        //TODO
+    }
```

Comment:
todo?

Reply:
yes I wanted to tackle that after the general idea was improved. This is a lot of work because you have to create customers and all the other things to compare two offers and or product items

Reply:
looks reasonable for me, so you mean adding the tests with this pr or with another? I think it would make sense to do it here/now

Reply:
yes within this pr. I did not want to put a lot of work into it though before the addition of the fields was approved